marbles/core/tests/test_main.py

```diff
@@ -215,9 +215,9 @@ def test_show_source(self):
         '''The source code should appear.'''
         if self.has_marbles:
             self.assertIn('Source', self.stderr)
-            self.assertIn('self.assertEqual(', self.stderr)
```

Comment:
This is kind of funny: `unittest` in python 3.8 now shows only the first line of the expression, rather than the last.